src/components/EventCard.tsx

```diff
@@ -42,11 +42,11 @@ const EventCard = ({
     <div className="flex my-8 md:my-10">
       <EventDate startTime={startTime} color={color} />
       <div className="flex-1 px-6 md:px-10">
-        <span className="font-sans">
+        <span className="font-sans md:text-base">
           {startTimeString + (endTime ? " to " + endTimeString : "")}
         </span>
-        <div className="flex flex-row justify-between items-start mb-4">
-          <div className="not-italic font-bold text-2xl leading-8 font-sans break-words w-4/5">
+        <div className="flex flex-row justify-between items-start mb-4 md:mb-0">
+          <div className="not-italic font-bold text-2xl leading-8 font-sans break-words w-4/5 md:pr-64 md:text-lg">
```

Comment:
Nit: Same here with the text size, I think it'd be fine to just keep the text size the same across breakpoints